.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "esp8266-oled-ssd1306"]
+	path = esp8266-oled-ssd1306
+	url = https://github.com/ClutchplateDude/esp8266-oled-ssd1306
```

Comment:
Instead of a submodule can we do a platformio git library? Like what we do for `avr-interrupt-stepper`. Submodules just tend to be a bit of a headache, so if we can avoid it that'd be nice

Reply:
I have no idea how to do that, I just read in a PR a while back that we should do submodules :-D . Anyway, my git knowledge is only rudimentary. Can you point at a doc that explains how to do that?

Reply:
Updated.